### PR TITLE
Added metrics for all time unduplicated users of prrr

### PR DESCRIPTION
**Pull request by shakalee14**
- added simple count query for users

### CODE REVIEW

File: server/queries/metrics.js [hunk 1]
```diff
@@ -55,6 +62,12 @@ export default class Metrics {
       .then(results => Number(results[0].count))
   }
 
+  totalUnduplicatedUsers(){
+    return this.usersQuery()
+      .select(this.knex.raw('COUNT(*)'))
```
Comment (deadlyicon) [inline server/queries/metrics.js]: can't you just do `.count('*')` ?

http://knexjs.org/#Builder-count

Comment (deadlyicon) [inline server/queries/metrics.js]: Can we try this?

```js
totalUsersCount(){
  return this.knex
    .table('users')
    .count('*')
    .then(result => Number(result[0].count))
}
```

File: server/queries/metrics.js [hunk 2]
```diff
@@ -38,12 +44,13 @@ export default class Metrics {
       return {
         week: this.week.format('YYYY-MM-DD'),
         totalCodeReviews: results[0],
-        totalCodeReviewsPerReviewer: results[1],
-        averageTimeForPrrrToBeClaimed: results[2],
-        averageTimeForPrrrToBeCompleted: results[3],
-        totalNumberOfProjectsThatRequestedReviews: results[4],
-        averageNumberOfReviewsRequestedPerProject: results[5],
-        prrrs: results[6],
+        totalUnduplicatedUsers: results[1],
```
Comment (deadlyicon) [inline server/queries/metrics.js]: rename to `totalUsersCount`

File: browser/components/pages/MetricsPage/index.js [hunk 3]
```diff
@@ -99,6 +99,10 @@ const Metrics = ({metrics}) => {
       <span>{metrics.totalCodeReviews}</span>
     </div>
     <div>
+      <strong>Total unduplicated Prrr users: </strong>
```
Comment (deadlyicon) [inline browser/components/pages/MetricsPage/index.js]: rename:
```html
<strong>Total users count: </strong>
```

File: browser/components/pages/MetricsPage/index.js [hunk 4]
```diff
@@ -99,6 +99,10 @@ const Metrics = ({metrics}) => {
...
+      <span>{metrics.totalUnduplicatedUsers}</span>
```
Comment (deadlyicon) [inline browser/components/pages/MetricsPage/index.js]:
```jsx
<span>{metrics.totalUsersCount}</span>
```

**Comment by shakalee14** [comment]
@deadlyicon

**Pull request merged**